content/en/docs/project-user-guide/grayscale-release/blue-green-deployment.md

```diff
@@ -7,4 +7,77 @@ linkTitle: "Blue-green Deployment"
 weight: 2130
 ---
 
-TBD
\ No newline at end of file
+
+The blue-green release provides a zero downtime deployment, which means the new version can be deployed with the old one preserved. At any time, only one of the version is active serving all the traffic, while the other one remains idle. If there is a problem with running, you can quickly roll back to the old version
+
+![](/images/service-mesh/blue-green-0.png)
+
+
+## Before you begin
```

Comment:
Replace the steps of the section **Before you begin** with this [doc](https://kubesphere.io/docs/pluggable-components/service-mesh/#enable-service-mesh-before-installation), you just need to add a reference link to it.